### PR TITLE
IA Animated gif support

### PR DESCRIPTION
**Pull request by MichaelButkovic**
@mparent61 @spra85 @benghaziboy

### CODE REVIEW

**Comment by benghaziboy** [comment]
:+1:

File: bulbs/instant_articles/parser.py [hunk 1]
```diff
@@ -31,7 +31,8 @@ def parse_betty(tag):
             tag.has_attr('data-image-id')):
         caption = tag.find('span', class_='caption')
         return {'betty': {'image_id': tag.attrs['data-image-id'],
-                          'caption': caption.text if caption else ''}}
+                          'caption': caption.text if caption else '',
+                          'format': tag.attrs['data-format']}}
```
Comment (mparent61) [inline bulbs/instant_articles/parser.py]: should add a `and tag.has_attr('data-format')` check to above conditional

**Comment by MichaelButkovic** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;jpQrRSJc&#x2F;29-animated-gifs-do-not-work"> Animated GIFs do not work</a>

File: tests/instant_articles/test_renderer.py [hunk 2]
```diff
@@ -24,6 +24,17 @@ def test_render_betty(self):
                 "<figure><img src=\"//images.onionstatic.com/onion/2349/16x9/1920.jpg\" /><figcaption>A really good caption</figcaption></figure>"
             )
 
+    def test_reender_betty_gif(self):
```
Comment (mparent61) [inline tests/instant_articles/test_renderer.py]: typo :)

**Comment by mparent61** [comment]
Otherwise LGTM

**Comment by spra85** [comment]
Also, otherwise LGTM after @mparent61's suggestion

**Pull request merged**